src/data/treasury-data.ts

```diff
@@ -76,3 +76,16 @@ export const TREASURIES = [
     href: 'https://opensea.io/assets/ethereum/0xc2c747e0f7004f9e8817db2ca4997657a7746928/6992',
   },
 ]
+
+export const chain = {
+  Eth: 'eth',
+  Matic: 'matic',
+}
+
+export const TREASURY_ADDRESSES = [
```

Comment:
thats not a treasury addresses, those are tokens addresses. Treasury address is where those tokens are stored and its not needed here

Reply:
also why those addresses are not just part of the TOKENS object that you need anyhow? splitting this way might lead into future bugs where one object is updated and the other dont.